From bc980fc7b09d047aa6a854ded5f122e0ea865147 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynhnguyen.xb@renesas.com>
Date: Wed, 14 Oct 2020 16:50:40 +0700
Subject: [PATCH 314/476] dt-bindings: PCI: rcar: Add device tree support for
 r8a7742

Add internal PCI bridge support for R8A7742 SoC. The Renesas RZ/G1H
(R8A7742) internal PCI bridge is identical to the R-Car Gen2 family.

No driver change is needed due to the fallback compatible value
"renesas,pci-rcar-gen2".

Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 Documentation/devicetree/bindings/pci/pci-rcar-gen2.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/pci/pci-rcar-gen2.txt b/Documentation/devicetree/bindings/pci/pci-rcar-gen2.txt
index 8ea3ec0..7857d87 100644
--- a/Documentation/devicetree/bindings/pci/pci-rcar-gen2.txt
+++ b/Documentation/devicetree/bindings/pci/pci-rcar-gen2.txt
@@ -6,7 +6,8 @@ AHB. There is one bridge instance per USB port connected to the internal
 OHCI and EHCI controllers.
 
 Required properties:
-- compatible: "renesas,pci-r8a7743" for the R8A7743 SoC;
+- compatible: "renesas,pci-r8a7742" for the R8A7742 SoC;
+	      "renesas,pci-r8a7743" for the R8A7743 SoC;
 	      "renesas,pci-r8a7744" for the R8A7744 SoC;
 	      "renesas,pci-r8a7745" for the R8A7745 SoC;
 	      "renesas,pci-r8a7790" for the R8A7790 SoC;
-- 
2.7.4

